bpf/lib/encrypt.h

```diff
@@ -68,7 +68,11 @@ do_decrypt(struct __ctx_buff *ctx, __u16 proto)
 		return CTX_ACT_OK;
 	}
 	ctx->mark = 0;
+#if 0 //ENABLE_HOST_REDIRECT? What is the correct ifdef here
```

Comment:
AFAIK, you want:
```suggestion
#ifdef ENABLE_ROUTING
```

`!defined(ENABLE_ROUTING)` <=> `enable-endpoint-routing=true`

Reply:
Thanks! That is what we want here.